ByBit.Net/Clients/SpotApi/v3/BybitClientSpotApiTradingV3.cs

```diff
@@ -93,15 +93,16 @@ public async Task<WebCallResult<BybitSpotOrderWrapper>> GetOpenOrdersAsync(strin
 
         #endregion
 
-        #region Get open orders
+        #region Get orders
 
         /// <inheritdoc />
-        public async Task<WebCallResult<BybitSpotOrderWrapper>> GetOrdersAsync(string? symbol = null, long? orderId = null, int? limit = null, long? receiveWindow = null, CancellationToken ct = default)
+        public async Task<WebCallResult<BybitSpotOrderWrapper>> GetOrdersAsync(string? symbol = null, long? orderId = null, int? limit = null, int? orderCategory = 0, long? receiveWindow = null, CancellationToken ct = default)
```

Comment:
I'd default it to null instead of 0. The line you added at 105 already handles that as well